Modules/Journey/tabs/QuestsByZone/QuestsByZoneTab.lua

```diff
@@ -128,7 +161,181 @@ _CreateZoneDropdown = function()
     return dropdown
 end
 
+_HandleAllZonesSelection = function()
```

Comment:
Do we actually need this code all over again? On a short look it looks very similar to the code in `QuestsByZone.lua`. If there are major differences, then maybe some parts that are the same can be extracted and be reused?